### DISCUSSION TITLE
Dotnet6 upgrade

### DISCUSSION THREAD

**Discussion by AhLay** [TimeWarpEngineering/blazor-state#268] [Q&A] [answered]
Hi,

first thank you for the efforts you are putting into this nice Librrary.
Do you have any plans for upgrading to dotnet6?

**Comment by StevenTCramer** [comment, marked as answer]
Absolutely. Within the month.

**Reply by AhLay** [reply]
ok, thx

**Reply by StevenTCramer** [reply]
You can use dotnet 5 libraries in dotnet 6 projects.  I am wondering if anything in particular was causing you a problem.  

I am updating the code base to C#10 now as I want to take advantage of some of the new features.  

But everything should work fine in a dotnet 6 app without these changes I am making.
